install/tools/ipa-pki-wait-running.in

```diff
@@ -74,9 +74,13 @@ def get_status(conn, timeout):
     """
     client = SystemStatusClient(conn)
     response = client.get_status(timeout=timeout)
-    root = ElementTree.fromstring(response)
-    status = root.findtext("Status")
-    error = root.findtext("Error")
+    status = None
+    error = None
+    try:
+        json_response = json.loads(response)
+        status = json_response['Response']['Status']
+    except Exception as e:
```

Comment:
Can a more specific exception be caught?

Reply:
Please see reply to bottom comment.